test/subset/data/Makefile.sources

```diff
@@ -18,6 +18,7 @@ TESTS = \
 	tests/layout.gpos4.tests \
 	tests/layout.gpos6.tests \
 	tests/layout.gpos8.tests \
+	tests/layout.gpos9.tests \
```

Comment:
Something should've be added to `test/subset/data/Makefile.am` list also.

Reply:
Ha. That's weird. Thanks for pointing out. The parallel list should be moved to `Makefile.sources` as well...

Reply:
Fixed